internal/pkg/fakeroot/fakeroot.go

```diff
@@ -388,10 +388,10 @@ func IsUIDMapped(uid uint32) bool {
 		sylog.Fatalf("could not retrieve user with UID %d: %s", uid, err)
 	}
 	e, err := config.GetUserEntry(userinfo.Name)
-	if err != nil {
+	if err != nil && strings.Contains(fmt.Sprintf("%v", err), "no mapping entry found") {
 		return false
 	}
-	if e.disabled {
+	if e != nil && e.disabled {
```

Comment:
the function will return true for non existing users

Reply:
I don't think so, because the previous check for "getPwUID()" causes a fatal error if the user does not exist.  Things would be quite messed up if the user does not exist.